proposals/veranon/veranon_proposal.md

```diff
@@ -0,0 +1,110 @@
+# VerAnon Proposal
+
+## Metadata
+- Proposal Author(s): [Alex Hache](https://github.com/ahache)
+- Chair(s): Alex Hache
+- Status: [Proposed]()
+- Since: November 20, 2024
+- Expected End Date: January 31, 2025
+- Version: 0.0.1
+
+[//]: # (Do not fill out below. To be filled out by chairs post-approval)
+- Start Date: <date>
+- URI: <uri>
+
+## Summary 
+
+VerAnon is a protocol for anonymous personhood verification using Semaphore, a zero-knowledge group membership protocol.
+- The protocol involves three key participants:
+  - Subjects: Individuals seeking to privately associate verified personhood with participating services
+  - Managers: Trusted entities that verify identities and maintain Semaphore groups
+  - Service Providers: Applications/services that require verified unique personhood
+- Subjects provide identity documents to Managers for verification 
+- Once verified, the Subject's Semaphore Identity Commitment is added to the Manager's Semaphore group
+- Managers handle identity verification, group administration, and registration records
+- Subjects can prove their group membership (personhood) to Service Providers anonymously and uniquely using zero-knowledge proofs generated by Semaphore circuits
+- Service Providers can select which Managers they trust for user personhood verification
+
+![VerAnon Protocol Diagram](./VerAnon-Diagram.png)
+
+
+## Motivation
+
+VerAnon represents one potential approach to addressing the goals outlined in the Personhood Credential Initiative (PHC). The primary motivation is to enhance trust in online spaces by:
+
+- Preventing bot accounts from being created at scale by enforcing unique use of personhood proofs within a service
+- Providing a more organic web experience for users
+- Protecting user privacy through ZKPs (anonymous verification and unlinkable use of proofs across different services)
+
+A larger motivation and vision for VerAnon is in expanding the use of existing identity systems. Where sybil resistance is easily achieved with low privacy guarantees, VerAnon would provide a solution to map a collection of identities uniquely and anonymously to Service Providers.
+
+## Technical Goals
+
+The primary goals are to implement the core protocol and supporting client applications. In addition, a pilot Manager and Service Provider would be useful to demonstrate the protocol.
```

Comment:
@ahache I had a conversation with @kimdhamilton about how the artifact is managed, and I would like to ask a quick question here: 

Would you be open to handling the artifact the following way:

1. The protocol specification can live in a DIF repository, so it has the IPR protections and continues to live in DIF as an open source project regardless of where you end up going personally later.
2. All code/artifacts, however you want to manage them outside the protocol specification, can live anywhere else, and may be included into DIF if you would like. 

Let me know if this works.

Reply:
This sounds great @andorsk 

I think it would be best if everything mentioned in the primary goals would be open and housed under DIF. It would probably also be useful to include the pilot Manager / Service Provider as a reference for anyone looking to use the protocol.